test.py

```diff
@@ -458,6 +462,153 @@ def test_schedule_purge(self):
         self.app.post('/VolumeDriver.Schedule', json.dumps(
             {'Name': name, 'Action': 'purge:2h:2h', 'Timer': 0}))
 
+    def test_synchronization(self):
+        """Check we can snapshot a volume
+        """
+        # create a volume with a file
+        name = PREFIX_TEST_VOLUME + uuid.uuid4().hex
+        path = join(VOLUMES_PATH, name)
+        remote_path = join(TEST_REMOTE_PATH, name)
+        # Prepare local btrfs subvolume
+        self.app.post('/VolumeDriver.Create', json.dumps({'Name': name}))
+        with open(join(path, 'foobar'), 'w') as f:
+            f.write('foobar')
```

Comment:
there is a new utility function ``self.create_volume_with_a_file(name)`` to lower the global number of lines